db/hybrid_logical_vector.go

```diff
@@ -92,18 +185,23 @@ func ParseDecodedVersion(versionString string) (version DecodedVersion, err erro
 	return version, nil
 }
 
-// String returns a Couchbase Lite-compatible string representation of the version.
-func (v DecodedVersion) String() string {
+// BlipString returns a Couchbase Lite-compatible string representation of the version.
+func (v DecodedVersion) BlipString() string {
 	timestamp := string(base.Uint64CASToLittleEndianHex(v.Value))
 	source := base64.StdEncoding.EncodeToString([]byte(v.SourceID))
 	return timestamp + "@" + source
 }
 
 // String returns a version/sourceID pair in CBL string format
-func (v Version) String() string {
+func (v Version) BlipString() string {
 	return v.Value + "@" + v.SourceID
 }
 
+// DeltaString returns a pv or mv delta format string representation of version
+func (v Version) DeltaString() string {
+	return v.SourceID + "@" + v.Value
```

Comment:
I don't think we want two different formats (value@source and source@value) - that's going to lead to confusion.  Can you talk to XDCR about standardizing on value@source?